### DISCUSSION TITLE
I want to migrate the utm virtual machine to vmware. What should I do?

### DISCUSSION THREAD

**Discussion by 3uph** [utmapp/UTM#5462] [General]

**Comment by startergo** [comment]
Install qemu with brew:

```
brew install qemu
```

Convert the qcow2 image to vmdk:

```
qemu-img convert -p -f qcow2 -O vmdk myimage.qcow2 newimage.vmdk
```

If it does not work directly you can try converting to raw first:

```
qemu-img convert -p -f qcow2 -O raw newimage.raw
qemu-img convert -p -f raw -O vmdk newimage.vmdk
```
Bear in mind the raw image may be 5 times bigger than the original so you need space during conversion.

This vmdk might not be able to be used on ESXi, but you can use it on VMware Workstation. To let it work on ESXi, you need to use vmkfstools to convert it again.

```
vmkfstools -i source.vmdk -d thin converted_source.vmdk
```

**Reply by 3uph** [reply]
Thank you very much!!

**Comment by macuserguru** [comment]
The other way import a VMware Fusion Image to UTM?
PS: I installed both UTM and VMware Fusion on my M1 Mac Mini. Now I want to move my FreeBSD 13 64-bit Arm VM to UTM without new installation. But I don't know what impact this might have on FreeBSD itself!

qemu are now installed by homebrew too

**Reply by Duoduo-chino** [reply]
![7261705923911_ pic](https://github.com/utmapp/UTM/assets/37808059/0f2a1239-8882-4b11-b775-7a286d6fd87d)
根据原始虚拟机的启动模式进行选择。

**Reply by pizzapi2012** [reply]
what @Duoduo-chino is saying that you need to turn on uefi
